pkgs/development/libraries/libprom/default.nix

```diff
@@ -0,0 +1,41 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv; # So it builds on Mac
```

Comment:
Sorry, I meant a comment explaining *why* this fixes building on darwin -- e.g. what goes wrong without it.

On Linux, `stdenv` is `gccStdenv`. While it's possible, I don't see why Darwin wouldn't be the same.

Reply:
> On Linux, `stdenv` is `gccStdenv`. While it's possible, I don't see why Darwin wouldn't be the same.

Darwing defaults to clang which for some reason doesn't compile this.

Reply:
Not sure either, but I was the one who wrote this part, having tested on MacOS and found that the default clang stdenv on Darwin did not work.

Reply:
Makes sense, thanks for the clarification (and for bearing with my nits).